test/quantization/core/test_quantized_op.py

```diff
@@ -3663,14 +3656,34 @@ def test_qlinear(self, batch_size, input_channels, output_channels, use_bias,
             X_fp32 = X_q.dequantize().to(dtype=torch.float)
             b_fp32 = b_q.dequantize().to(dtype=torch.float) if use_bias else None
             Y_fp32_ref = F.linear(X_fp32, W_fp32, b_fp32)
-            if use_relu:
+            if post_op == 'relu':
                 Y_fp32_ref[Y_fp32_ref < 0.0] = 0.0
+            elif post_op == 'leaky_relu':
+                Y_fp32_ref = F.leaky_relu(Y_fp32_ref, **post_op_kwargs)
             Y_q_ref2 = torch.quantize_per_tensor(
                 Y_fp32_ref, Y_scale, Y_zp, dtype)
             # Assert equal
             np.testing.assert_array_almost_equal(
                 Y_q_ref2.int_repr().numpy(), Y_q.int_repr().numpy(), decimal=decimal_val)
 
+    """Tests the correctness of the quantized linear and linear_relu op."""
+    @override_qengines
+    def test_qlinear(self):
```

Comment:
looks like this is qlinear + qlinear relu? I feel we can split this to two tests? we would like to make each test a unit test I think

Reply:
Ok I will do that.

Reply:
I have split it to two separate tests.